api/fs/root.go

```diff
@@ -0,0 +1,55 @@
+package apifs
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/puppetlabs/wash/plugin"
+)
+
+// Root of the GCP plugin
+type Root struct {
+	dir
+}
+
+// Init for root
+func (r *Root) Init(cfg map[string]interface{}) error {
+	var basepath string
+	if pathI, ok := cfg["basepath"]; ok {
+		basepath, ok = pathI.(string)
+		if !ok {
+			return fmt.Errorf("local.basepath config must be a string, not %s", pathI)
+		}
+	} else {
+		basepath = "/tmp"
+	}
+
+	finfo, err := os.Stat(basepath)
+	if err != nil {
+		return err
+	}
+
+	r.dir.fsnode = newFSNode(finfo, basepath)
+
+	r.EntryBase = plugin.NewEntry("local")
+	r.DisableDefaultCaching()
+	return nil
+}
+
+// Schema returns the root's schema
+func (r *Root) Schema() *plugin.EntrySchema {
+	return plugin.
+		NewEntrySchema(r, "local").
+		SetDescription(rootDescription).
+		IsSingleton()
+}
+
+var _ = plugin.Root(&Root{})
+
+const rootDescription = `
```

Comment:
Per https://github.com/puppetlabs/wash/pull/664#discussion_r358520344, you could probably capture the config in the LOCALFS environment variable.

Reply:
So some of the discussion in https://github.com/puppetlabs/wash/issues/438 pointed to making apifs just another plugin. That's why I'm implementing it so closely to one.

Reply:
You could still keep `Init` as-is, you'd just create the config in `server.go` from the env. var.

Reply:
Done.